akka-typed/src/main/scala/akka/typed/ActorContext.scala

```diff
@@ -62,12 +62,12 @@ trait ActorContext[T] {
    * Create a child Actor from the given [[Props]] under a randomly chosen name.
    * It is good practice to name Actors wherever practical.
    */
-  def spawnAnonymous[U](props: Props[U]): ActorRef[U]
+  def spawnAnonymous[U](behavior: Behavior[U], dispatcher: DispatcherSelector = DispatcherDefault, mailboxCapacity: Int = 1000000): ActorRef[U]
```

Comment:
If we retain Props then we must pay the allocation overhead for them. One thing we could do in order to retain flexibility is to make the dispatcher selector more generic and extensible, using an internal list for composition. This means that we could also fold the mailbox capacity into the same mechanism, reducing the boilerplate. I’ll whip up a draft tonight.